assets/src/scripts/components/hydrograph/drawingData.js

```diff
@@ -83,18 +84,19 @@ export const allPointsSelector = createSelector(
  * @param {String} tsKey
  * @return {Object} of keys are tsId, values are Array of point Objects
  */
-export const pointsByTsKeySelector = memoize(tsKey => createSelector(
+export const pointsByTsKeySelector = memoize(tsKey => memoize(period => createSelector(
```

Comment:
The new memoization library we're using - fast-memoize - should support multiple arguments.

Reply:
I'll take a look.